pcawg_metadata_parser/generate_release.py

```diff
@@ -37,34 +37,91 @@
                         },
                         {
                           "terms":{
-                            "flags.is_sanger_variant_calling_performed":[
+                            "flags.is_normal_specimen_aligned":[
+                              "T"
+                            ]
+                          }
+                        },
+                        {
+                          "terms":{
+                            "flags.are_all_tumor_specimens_aligned":[
                               "T"
                             ]
                           }
+                        }                    
+                      ],
+                      "should": [
+                        {
+                           "terms":{
+                              "flags.is_sanger_variant_calling_performed":[
+                                 "T"
+                              ]
+                           }
+                        },
+                        {
+                           "bool":{
+                               "must":[
+                                      {
+                                       "terms":{
+                                          "flags.is_dkfz_embl_variant_calling_performed":[
+                                             "T"
+                                          ]
+                                        }
+                                      },
+                                      {
+                                         "term":{
+                                            "flags.is_dkfz_variant_calling_performed":[
+                                               "F"
+                                            ]
+                                         }
+                                      },
+                                      {
+                                         "term":{
+                                            "flags.is_embl_variant_calling_performed":[
+                                               "F"
+                                            ]
+                                         }
+                                      } 
+                                ]
+                           }
```

Comment:
These filtering criteria ideally are optional and can be specific by command line arguments. The reason is that we will use this script to generate different kind of summary reports/lists (other than just a specific PCAWG data release, which does need more stringent filtering), for a general overview report, we'd like to have minimum filtering.

Maybe we merge for now, and add this feature later.